lib/releases.js

```diff
@@ -80,74 +79,84 @@ const contributorsSentence = ({ commits, pullRequests }) => {
 }
 
 const getCategoriesConfig = ({ config }) => {
-  const categoriesConfig = config.categories.reduce((acc, category) => {
-    const labels = category.label ? [category.label] : category.labels
-    labels.forEach(label => (acc[label] = category))
-    return acc
-  }, {})
-
-  const orderedCategories = [
-    UNCATEGORIZED,
-    ..._.flatMap(config.categories, category =>
-      category.label ? category.label : category.labels
-    )
-  ]
-
-  return [categoriesConfig, orderedCategories]
+  const categoriesConfig = [{ title: UNCATEGORIZED }, ...config.categories]
+
+  return categoriesConfig.map(category => {
+    if (!category.labels) category.labels = []
+    if (category.label) {
+      // treat label as labels
+      category.labels.push(category.label)
+      delete category.label
+    }
+    return { ...category, pullRequests: [] }
+  })
```

Comment:
This should probably be part of the config validation. Can you move this to `config.js` or `schema.js` as `validateCategoriesConfig` or `normalizeCategoriesConfig`?

Reply:
Yes schema can totally take care of most of it.

Now the function only handles unwrapping categories into allCategoryLabels, orderedCategories

Reply:
The label to labels and making sure it is an array has moved to schema.json